### PR TITLE
Add Hindi translations

### PR DESCRIPTION
**Pull request by waltzaround**
Adds Hindi translations

### CODE REVIEW

File: src/translations/locales/common_hi-HI.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      "sectionHeader": "अन्य टीकाकरण स्थान  <0>- आज ही खोलें </0>", 
+      "disclaimer": {
+        "title": "यह स्थान जनता द्वारा सबमिट किया गया है",
+        "message": " यहां दी गई जानकारी 100% सटीक नहीं हो सकती है। कृपया स्थान की वेबसाइट देखें"
+      }
+    }
+  },
+  "calendar": {
+    "month": "{{monthString, MMMM yyyy}}",
+    "numberOfAppointments": "{{sumAvailableTimes}} available",
```
Comment (noway) [inline src/translations/locales/common_hi-HI.json:70]: this one missed 🙏🏻

File: src/translations/locales/common_hi-HI.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      },
+      "availableSlots": "बुक करने के लिए उपलब्ध",
+      "noBookingsAvailable": "इस दिन कोई बुकिंग उपलब्ध नहीं है", 
+      "notBMV": "यह स्थान उपयोग नहीं करता  <0>bookmyvaccine.nz</0>  और खाली स्लॉट नहीं हो सकते हैं"
+    }
+  },
+
+  "footer": {
+    "message": "अगर इस वेबसाइट ने आपकी सहायता की है, तो कृपया इसे शेयर करें",
+    "addressFinderLinkCopy": "Thanks to <0>AddressFinder</0> for free-of-charge address lookup",
```
Comment (noway) [inline src/translations/locales/common_hi-HI.json:87]: This one missed too 🙏🏻

File: src/translations/locales/common_hi-HI.json [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    "links": {
+      "contactUs": " संपर्क करें",
+      "rawData": "Raw Data",
+      "sourceCode": "Source Code",
+      "roadmap": "Roadmap"
```
Comment (noway) [inline src/translations/locales/common_hi-HI.json:92]: and these 3 🙏🏻

**Review by noway — CHANGES_REQUESTED** [review]
if we can add these missing ones, would be amazing. allg if not!

**Comment by olafwrieden** [comment]
Please import and add locale to `locale.ts`

**Comment by noway** [comment]
image
Does this look all good or is it missing a space? Can a native speaker confirm.

File: src/translations/locales/common_hi-HI.json [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    "loading": "लोड हो रहा है",
+    "getDirections": "दिशा - निर्देश प्राप्त करें",
+    "makeABooking": "बुकिंग करें",
+    "kmAway": "{{distance}}किमी दूर",
+    "lastUpdated": "आखरी अपडेट{{updatedAt}}",
+    "cancel": "रद्द करें",
+    "website": "वेबसाइट",
+    "address": "पता",
+    "noResultsNotInNZ": "No vaccination sites found for this search query. Are you in New Zealand?",
+    "noResultsInRadius": "No vaccination sites found for this search query. Try a different kilometer radius."
```
Comment (noway) [inline src/translations/locales/common_hi-HI.json:16]: This two are missing 🙏🏻

File: src/translations/locales/common_hi-HI.json [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+      "5km": "5 . के भीतर किमी",
+      "10km": "10 . के भीतर किमी",
+      "25km": "25 . के भीतर किमी",
+      "50km": "50 . के भीतर किमी",
+      "100km": "100 . के भीतर किमी"
+    }
+  },
+  "walkins": {
+    "sectionHeader": "आज खुला है",
+    "openString": "Open {{openTimeString}}",
```
Comment (noway) [inline src/translations/locales/common_hi-HI.json:43]: This one missing too 🙏🏻

File: src/translations/locales/common_hi-HI.json [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    "noticeList": {
+      "title": "टीका कैसे लगवाएं",
+      "text": "टीकाकरण की कतार हो सकती है, और कुछ केंद्रों में आज अतिरिक्त टीके उपलब्ध नहीं हो सकते हैं - यदि आप कर सकते हैं तो आगे फोन करें!"
+    },
+    "walkinAwailable": "<0 />बिना नियोजित भेंट टीकाकरण उपलब्ध",
+    "driveThroughAvailable": "<0 /> ड्राइव करते हुए टीकाकरण उपलब्ध",
+    "phone": "फ़ोन",
+    "hours": "घंटे",
+    "closed": "बंद",
+    "publicHolidays": "Public Holidays",
```
Comment (noway) [inline src/translations/locales/common_hi-HI.json:58]: This one missing 🙏🏻

File: src/translations/locales/common_hi-HI.json [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+        "title": "कैसे बुक करें",
+        "stepOne": "नीचे दी गई सूची से स्थान और समय खोजें",
+        "stepTwo": "पर क्लिक करें बुकिंग करने के लिए, यह आपको ले जाएगा <0>bookmyvaccine.nz</0>",
+        "stepThree": "अपना विवरण दर्ज करें",
+        "stepFour": "पूछे जाने पर, अपना इच्छित स्थान और समय खोजें"
+      },
+      "availableSlots": "बुक करने के लिए उपलब्ध",
+      "noBookingsAvailable": "इस दिन कोई बुकिंग उपलब्ध नहीं है",
+      "notBMV": "यह स्थान उपयोग नहीं करता  <0>bookmyvaccine.nz</0>  और खाली स्लॉट नहीं हो सकते हैं",
+      "backToCalendar": "Back to Calendar"
```
Comment (noway) [inline src/translations/locales/common_hi-HI.json:83]: this one 🙏🏻

**Comment by waltzaround** [comment]
On standby until we get the remaining translations

**Review by noway — APPROVED** [review]
LGTM SHIP IT

**Pull request merged**